frontend/src/app/AppHeader.js

```diff
@@ -23,13 +23,28 @@ function AppHeader(props) {
                     </span>
                 </FlexboxGrid.Item>
                 <FlexboxGrid.Item>
-                    <SearchAndFilter/>
+                    <FlexboxGrid>
+                        <SearchAndFilter/>
+                        <BibButton setShowBib={props.setShowBib}/>
+                    </FlexboxGrid>
                 </FlexboxGrid.Item>
             </FlexboxGrid>
         </Navbar>
     );
 }
 
+class BibButton extends React.Component {
+    render() {
+        return(
+            <FlexboxGrid>
+                <FlexboxGrid.Item>
+                    <IconButton onClick={() => this.props.setShowBib(true)} icon={<Icon icon="book"/>}/>
+                </FlexboxGrid.Item>
+            </FlexboxGrid>
```

Comment:
```suggestion
            <IconButton onClick={() => this.props.setShowBib(true)} icon={<Icon icon="book"/>}/>
```
I don't think you need the wrapping flexboxgrid here